src/include/tcop/tcop.h

```diff
@@ -22,7 +22,7 @@
 #include "common/statement.h"
 #include "concurrency/transaction.h"
 #include "executor/plan_executor.h"
-#include "optimizer/abstract_optimizer.h"
```

Comment:
I don't know if we still need AbstractOptimizer. On my opinion, we use AbstractOptimizer because we can easily change optimizer from simple to not simple optimizer mode. And in this part, pmenon doesn't allow me declare txn in abstract_optimizer, but I still need use txn, so I can only change it to optimizer.